test/resources/gatling.conf

```diff
@@ -0,0 +1,127 @@
+#########################
```

Comment:
Maybe cut this down to just the settings we need? If it's idiomatic for Gatling projects to commit the whole thing no big deal